src/base/bittorrent/session.cpp

```diff
@@ -4480,54 +4480,63 @@ void Session::startUpTorrents()
         const lt::info_hash_t infoHash = (resumeData.ltAddTorrentParams.ti
                                           ? resumeData.ltAddTorrentParams.ti->info_hashes()
                                           : resumeData.ltAddTorrentParams.info_hashes);
-        if (infoHash.has_v1() && infoHash.has_v2())
+//        const bool isHybrid = infoHash.has_v1() && infoHash.has_v2();
+        const auto torrentIDv2 = TorrentID::fromInfoHash(infoHash);
+        const auto torrentIDv1 = TorrentID::fromInfoHash(lt::info_hash_t(infoHash.v1));
+        if (torrentID == torrentIDv2)
         {
-            const auto torrentIDv1 = TorrentID::fromInfoHash(lt::info_hash_t(infoHash.v1));
-            const auto torrentIDv2 = TorrentID::fromInfoHash(infoHash);
-            if (torrentID == torrentIDv2)
+            if (indexedTorrents.contains(torrentIDv1))
             {
-                if (indexedTorrents.contains(torrentIDv1))
+                // if we has no metadata trying to find it in alternative "resume data"
```

Comment:
This sentence is confusing and doesn't seem to be worded correctly.

**These would make more sence:**
if we don't have metadata, try to find it in ...
if we do have the metadata, try to also find it in ...
if there's no metadata, try to find it in ...
if qBitorrent doesn't have the torrents metadata, try to find it in ...